integration/__tests__/app.tests.ts

```diff
@@ -47,17 +47,23 @@ describe("Lens integration tests", () => {
       await clickWhatsNew(app)
     })
 
-    // Todo figure out how to access main menu to get these to work
-    it.skip('shows "add cluster"', async () => {
```

Comment:
I would say that we should keep the other test since it verifies a different code path. (unless the previous test just doesn't work)

Reply:
I removed it primarily because it doesn't work (which is why it was skipped). On top of this the `keys()` method is deprecated. I looked into the alternative using Actions but it is more involved. The scope of this PR is just to get the menu-invoked functionality testable.

You are right though, we should exercise the menus via accelerators as well as mouse clicks. For another PR...